src/components/ToDoItem/ToDoItem.js

```diff
@@ -2,10 +2,10 @@ import './ToDoItem.css';
 
 import { deleteTaskStorage, updateTaskInStorage } from '../../helpers/storage';
 
-export const ToDoItem = ({ message, id, completed = false }) => {
+export const ToDoItem = ({ message, id, completed = false, createdAt }) => {
```

Comment:
here you need to desctructure rest of parameters as well in order to display them from localstorage, in other case there will always be equal to ''

Reply:
You mean that I should pass all object properties that are created by function createTaskStorage as parameters in function ToDoItem ?

Reply:
.